.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
in your .gitignore file you should have .idea/* line so that you don't get these files